includes/admin/home/tasks/task/class-sensei-home-task-sell-course-with-woocommerce.php

```diff
@@ -9,6 +9,8 @@
 /**
  * Sensei_Home_Task_Sell_Course_With_WooCommerce class.
  *
+ * @deprecated $$next-version$$ We're now showing the "Sell your course with Sensei Pro" (Sensei_Home_Task_Pro_Upsell) task instead of this task.
```

Comment:
WDYT of adding a constructor here calling the `_deprecated_function`. So if something is still using it will add the warning to the logs.